core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
@pranavsid98 @seanlip tl;dr - job creations is synchronous while classifier creation is asynchronous by design. 

Firstly, we will be creating only training jobs in this flow. So, the train() method should - check the training conditions, and, create a new training job in the respective job table (but before that we need to implement storage & domain level classes for job table). This (creating jobs) should be done spontaneously so making train() async doesn't really give you any advantage. To answers @seanlip's question, the VM approach is async by design, so, the creator will have to wait for the classifier to be generated. We haven't really thought about what the UX for a creator is going to be like, but, we will make sure that it is simple and easy for him/her to use.